tests/rules/import-alias.test.ts

```diff
@@ -123,33 +116,77 @@ function runTests(platform: "win32" | "posix") {
                 ],
             },
 
-            // relative path overridden for root and exports from parent module
```

Comment:
Hmm, I might be wrong but the diff makes it look like the tests were changed? I'd suggest doing strict additions of tests instead of modifications of existing tests.

Reply:
yeah, i remove the tests and add them again to have. Rewriting a bit what was making more sense for me since you pointed out in https://github.com/Limegrass/eslint-plugin-import-alias/pull/8#discussion_r1493428431 that i should put more focus on test